charts/s1-agent/templates/NOTES.txt

```diff
@@ -32,6 +32,11 @@ Configuration options used to deploy:
 {{- else }}
 * NO proxy will be used between AGENTS and MANAGEMENT.
 {{- end }}
+{{- if .Values.configuration.dv_proxy }}
```

Comment:
I think you can squash all 3 commits to one commit and name it:
charts: templates: daemonset: add dv proxy config